docs/tarball.md

```diff
@@ -12,14 +12,23 @@ oci_tarball(
 )
 ```
 
-and then run it in a container like so:
+and then load and run it in a container like so:
 
 ```
-bazel build //path/to:tarball
-docker load --input $(bazel cquery --output=files //path/to:tarball)
-docker run --rm my-repository:latest
+bazel run :tarball
+docker run --rm "my-repository:latest"
 ```
 
+By default, the generated script will attempt to use Docker to load the image. If you need to use a runtime other than docker, you can replace the entire script template:
```

Comment:
can update this? Now that it searches for a command, most users wouldn't need `run_template` anyway, it feels like more of a "break the glass" escape hatch now